WebAPI/Services/BankStatmentReader.cs

```diff
@@ -141,14 +138,18 @@ private decimal GetAmout(decimal sum, string debitOrCredit)
             ///Transaction type,Date,Time,Amount,Equivalent,C/D,Orig. amount,Orig. currency,Document number,Transaction ID,
             ///Customer‘s code in beneficiary IS,Payment code,Payment details,Counterparty BIC,Counterparty Designation of counterparties credit institution,
             ///Counterparty Account number,	Counterparty Designation,Counterparty Reg No.,Counterparty Customer‘s code in payer IS,Ultimate Payer Account number
-
-            return csv.TryGetField("Orig. amount", out decimal sum) && csv.TryGetField("C/D", out string debitOrCredit)
+            var a = csv.TryGetField("Orig. amount", out string adebitOrCredit);
+            var b = csv.TryGetField("Equivalent", out string asuma);
+            var c = csv.TryGetField("Time", out string adata);
+            var d = csv.GetField("Counterparty Account number");
+            var e = csv.GetField("Counterparty BIC");
```

Comment:
Why do you use these names? Wouldn't it be better to use discards?

Reply:
They were for testing purposes, I'll delete them